src/test/java/guitests/guihandles/TodoListViewHandle.java

```diff
@@ -88,36 +87,59 @@ public int getFirstTaskIndex(ImmutableTask task) {
     /**
      * Gets a {@link TaskCardViewHandle} object with the position {@code listIndex} located in the to-do list view.
      * Guaranteed unique result for identical tasks.
-     *
+     * Inherits the behaviour from {@link #getTaskCardViewNode(int)}.
      *
      * @param listIndex Index of the {@link #getImmutableTaskList()}.
      * @return An instance of the handle.
      */
     public TaskCardViewHandle getTaskCardViewHandle(int listIndex) {
-        Optional<Node> taskCardNode = getTaskCardViewNode(listIndex);
-        return (taskCardNode.isPresent()) ? new TaskCardViewHandle(guiRobot, primaryStage, taskCardNode.get())
+        Node taskCardNode = getTaskCardViewNode(listIndex);
+        return (taskCardNode != null) ? new TaskCardViewHandle(guiRobot, primaryStage, taskCardNode)
```

Comment:
Would suggest not using a ternary operator here because this code is actually slightly complicated - it's doing two things at once, which makes it a bit hard to read